Sounds and Hud

emmanuelOpenForge opened the pull request:
# Pull request type

-   [x] Bugfix
-   [x] Feature
-   [ ] Code style update (formatting, renaming)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] Documentation content changes
-   [ ] E2E Test(s)
-   [ ] Other (please describe):

# What is the current behavior?
No music on damage or powers
No design on points
Old design on pause modal
# What is the new behavior?
- Added music for powerup and damages
- Added new design for hud of points
- Improved modal style
# Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

# Other information

Ricardo385 commented on libs/shared/phaser-singleton/src/lib/phaser-singleton.module.ts:
```diff
@@ -77,7 +77,7 @@ export class PhaserSingletonService {
                     parent: 'stage-main',
                     scale: {
-                        mode: Phaser.Scale.FIT,
+                        // mode: Phaser.Scale.FIT,
```
Is this commented on purpose? If not, could you uncomment this line, please?

Ricardo385 reviewed (changes requested):
Just a minor comment

Ricardo385 reviewed (approved):
nice

emmanuel-Andela committed:
Sounds and Hud
- Added music for powerup and damages
- Added new design for hud of points
- Improved modal style

emmanuel-Andela committed:
Sounds and Hud
- Added music for powerup and damages
- Added new design for hud of points
- Improved modal style
